core/android/contact/build.gradle.kts

```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 android {
-    namespace = "net.thunderbird.core.contact"
+    namespace = "net.thunderbird.core.android.contact"
```

Comment:
So this means this is an android-specific module example? I think we need to have some documentation on when to use `android` in the package name, and at what level. Otherwise I guess we'll have a mix of some namespaces being with `android` and some being without.

Reply:
Yes, it contains only android specific code, hence the package name. I'm preparing a documentation for the overall structure.